inst/tests/tests.Rraw

```diff
@@ -17577,3 +17577,10 @@ vr = "Species"
 IDT[, virginca := get(vr) == "virginica"]
 ans = data.table(round = c(3, 3, 3, 2, 2, 4, 2, 4), k = c(6, 7, 8, 5, 7, 7, 6, 8), kar = structure(c(3L, 3L, 3L, 3L, 3L, 3L, 3L, 3L), .Label = c("setosa", "versicolor", "virginica"), class = "factor"), N = c(24L, 14L, 4L, 1L, 1L, 1L, 3L, 2L))
 test(2184.5, IDT[(virginca), .N, by = .(round(Sepal.Width), k = round(Sepal.Length), kar = get(vr))] , ans)
+
+# non-ASCII plain symbol in by, #4708
+## test is wonky to be able to use a non-ASCII symbol in a script where ASCII is required for CRAN
```

Comment:
No words like `wonky` please. i) it probably doesn't translate well, ii) different cultures even within English have different interpretations, iii) similar to phrases 'doesn't work' and 'as expected', it lacks detail.
Suggest :
```
# eval(parse()) because this file needs to be ASCII on CRAN
```
The wonkiness is replaced by what is wonky instead; i.e. the `eval(parse())`.  We already have lots of `\u` usage in this file so that isn't the part that is wonky.

(In the early days of data.table I remember confusion from a user over my use of `like`. I remember explaining that like does not mean "exactly the same as'.  I meant 'like' as in 'similar to'.  If I had meant 'exactly the same as' I would have said so.  Anyway, over time, I have learnt that excruciating correct precision is best where possible.)

Reply:
Ack, thanks!